Source/WebKit/WebProcess/Plugins/PluginView.cpp

```diff
@@ -840,6 +859,11 @@ void PluginView::redeliverManualStream()
         manualLoadDidFinishLoading();
 }
 
+CheckedPtr<RenderEmbeddedObject> PluginView::checkedRenderer() const
+{
+    return dynamicDowncast<RenderEmbeddedObject>(m_pluginElement->renderer());
```

Comment:
Can the renderer be of any other type? If not, you could consider calling `checkedDowncast<>()` instead.

Reply:
`HTMLPlugInElement` uses `dynamicDowncast<RenderEmbeddedObject>` everywhere, so it's hard to know.